src/main/ansible/roles/testdata/tasks/main.yml

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+- name: Ensure /etc/ansible/facts.d exists
+  file:
+    path: /etc/ansible/facts.d
+    recurse: yes
+    state: directory
+
+- name: Copy facts-scripts to server
+  copy:
+    src: conditions.fact
+    dest: /etc/ansible/facts.d/
+    mode: "u+x"
+  register: result
+
+- name: Reload ansible_local
+  setup: filter=ansible_local
+  when: result.changed
+
+- name: Create link to test data
+  file:
+    src: /vagrant/src/test/resources/bags/basic-sequence-unpruned-with-refbags
+    dest: /home/vagrant/test-data
+    state: link
+
+- name: Create archive folder
+  file:
+    path: /home/vagrant/test-zips
+    state: directory
+    mode: 0777
+  when: ansible_local.conditions.test_zips_do_not_exist
+
+- name: Archive test data
```

Comment:
@rvanheest : Why this step?

Reply:
I think I see why you included this: so that POST-ing the bags through the web-interface can be done immediately. I think we can leave this out. When needed this is a trivial step with `zip -r deposit.zip srcdir`. 

(Also, the description 'Archive' is a bit confusing in the context of archiving software.)

Reply:
The reasoning for this is because we start of with an empty bagstore. Therefore the first step will _always_ be to add bags to a store. So _always_ you're going to do this `zip` command. Therefore I thought I might as well create the zips automagically.

Reply:
Mmm, given the comment below about not having just one set of test bags, it will probably make more sense indeed to do the zipping manually. Otherwise we would have to create zips for all bags in all the sequences. I'll remove it and see how we do without the initial zips.